public/src/modules/navigator.js

```diff
@@ -586,8 +594,13 @@ define('navigator', ['forum/pagination', 'components', 'hooks', 'alerts'], funct
 
 				navigator.scrollActive = false;
 				highlightPost();
-				$('body').scrollTop($('body').scrollTop() - 1);
-				$('html').scrollTop($('html').scrollTop() - 1);
```

Comment:
@barisusakli these two lines I was unsure of. They were last changed 8 years ago, but it did not seem to affect the scrolling and calculation logic when removed. Let me know if I should add them back.

Reply:
The removed lines function as a hack to hide the auto-hiding navbar used in Persona. I'd prefer not to introduce theme-specific code here.

In a new commit I've added client-side hooks for Persona to hook into.